doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -240,63 +275,11 @@ Error list (HTTP response code in paranthesis):
 * LengthRequired (411). Zero/No Content-Length in PUT/POST/PATCH request
 * RequestEntityTooLarge (413). Payload too large
 * UnsupportedMediaType (415). Request content type is not supported.
-* InvalidModification (422). Some piece of information is missing in payload.
-
-
-# Group NGSIv2
-
-## API Entry Point [/v2]
-
-### Retrieve Entry Points [GET /v2]
-
-This resource does not have any attributes. Instead it offers the initial
-API affordances in the form of the links in the JSON body.
-
-It is recommended to follow the “url” link values,
-[Link](https://tools.ietf.org/html/rfc5988) or Location headers where
-applicable to retrieve resources. Instead of constructing your own URLs,
-to keep your client decoupled from implementation details.
-
-+ Response 200 (application/json)
-
-        {
-            "entities_url":      "/v2/entities",
-            "types_url":         "/v2/types",
-            "subscriptions_url": "/v2/subscriptions",
-            "registrations_url": "/v2/registrations"
-        }
-
-## Entities [/v2/entities{?limit,offset,options,type,id,idPattern,q,geometry,coords,attrs}]
-
-### List entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,q,geometry,coords,attrs}]
-
-Retrieves a list of entities which match criteria defined by the following parameters: `id`, `type`, `idPattern`, `q`,
-`geometry` and `coords` attribute (see below for a detailed description of these parameters). A given entity
-have to match all the criteria to be retrieved (i.e. criteria are combined in a logical AND way).
-
-<table style="width: 100%; margin: 12px 0 0 0;">
-<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
-<tr><td colspan="2">none</td></tr>
-<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
-<tr>
-  <td style="padding-right: 40px; width: 140px;">id</td><td>A comma separated list of elements.
-  Retrieve entities which ID match one of the elements in the list.</td>
-</tr>
-<tr>
-  <td style="padding-right: 40px; width: 140px;">type</td><td>A comma separated list of elements.
-  Retrieve entities which type match one of the elements in the list.</td>
-</tr>
-<tr>
-  <td style="padding-right: 40px; width: 140px;">idPattern</td><td>A correctly formated regular
-  expression. Retrieve entities which ID matches the regular expression.</td>
-</tr>
-<tr><td style="padding-right: 40px; width: 140px;">q</td><td>
 
-<p>A query expression, composed of a list of statements separated by <code>;</code>, i.e.</p>
+## Simple Query Language
 
-<p><code>
-q=statement;statements;statement
-</code></p>
+The Simple Query Language provides a simplified syntax to retrieve entities which match a set of conditions. The
+supported syntax is:
 
 <p>Each statement express a matching condition. The result of the query is the match on each one of the statements, e.g. a
```

Comment:
Before expalining what an statement is, you have to explain that the syntax for the `q` parameters is `statement1;statement2;...;statementN` (that was already in the origininal .apib, but I guess it gets lost on the cut-and-paste operation).